dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/ProcessingManager.java

```diff
@@ -87,6 +87,14 @@ private ProcessingManager(Configuration config) throws InvalidConfigurationExcep
         final FunctionProcessor sccp = constantPropagation ? SparseConditionalConstantPropagation.fromConfig(config) : null;
         final FunctionProcessor dce = performDce ? DeadAssignmentElimination.fromConfig(config) : null;
         final FunctionProcessor removeDeadJumps = RemoveDeadCondJumps.fromConfig(config);
+        final ProgramProcessor repeatableAfterTearing = ProgramProcessor.fromFunctionProcessor(
+                FunctionProcessor.chain(
+                        performAssignmentInlining ? AssignmentInlining.newInstance() : null,
+                        sccp,
+                        dce,
+                        removeDeadJumps
+                ), Target.THREADS, true
+        );
```

Comment:
I don't like the variable name. The set of passes here has nothing to do with `Tearing`. Just give it a simple name like `standardSimplifications`.